Composer/packages/server/src/models/settings/defaultSettingManager.ts

```diff
@@ -121,6 +121,9 @@ export class DefaultSettingManager extends FileSettingManager {
         maxImbalanceRatio: -1,
       },
       skill: {},
+      skillConfiguration: {
+        allowedCallers: ['*'],
```

Comment:
i don't know why this is here. this isn't an appropriate default parameter

Reply:
I see,  `allowedCallers`  has been removed now